bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
I'd prefer `.copy()` so we can allow document-specific subscriptions like `self.default_subs = {'start': cb}`